test/unit/chunkedstream_spec.js

```diff
@@ -0,0 +1,713 @@
+/* -*- Mode: Java; tab-width: 2; indent-tabs-mode: nil; c-basic-offset: 2 -*- */
+/* vim: set shiftwidth=2 tabstop=2 autoindent cindent expandtab: */
+/* globals expect, it, describe, beforeEach, Stream, ChunkedStreamContinuous,
+   ChunkedStreamFragmented, Dict,  MissingDataException,
+   ALLOCATE_NO_CHUNKS_SIZE, ChunkedStreamBase, jasmine */
+
+'use strict';
+
+describe('stream', function () {
+
+  var arrayComparerThatFixesJasmineIssue786 = function (first, second) {
```

Comment:
Jasmine did fix the issue but it seems they didn't release a new version with this fix yet.